README_EN.md

```diff
@@ -0,0 +1,233 @@
+# Rexxar iOS
+
+[![Test Status](https://travis-ci.org/douban/rexxar-ios.svg?branch=master)](https://travis-ci.org/douban/rexxar-ios)
+[![IDE](https://img.shields.io/badge/XCode-8-blue.svg)]()
+[![iOS](https://img.shields.io/badge/iOS-7.0-green.svg)]()
+[![Language](https://img.shields.io/badge/language-ObjC-blue.svg)](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Introduction/Introduction.html)
+
+**Rexxar** is a hybrid library for mobile application development. Now it supports iOS and Android platform. `Rexxar iOS` is Rexxar's Container in iOS.
```

Comment:
... is a cross-platform library for hybrid mobile application. It supports both iOS and Android.